themes/classic/templates/catalog/_partials/quickview.tpl

```diff
@@ -30,13 +30,18 @@
               <form action="{$urls.pages.cart}" method="post" id="add-to-cart-or-refresh">
                 <input type="hidden" name="token" value="{$static_token}">
                 <input type="hidden" name="id_product" value="{$product.id}" id="product_page_product_id">
+                <input type="hidden" name="id_customization" value="{$product.id_customization}" id="product_customization_id">
                 {block name='product_variants'}
                   {include file='catalog/_partials/product-variants.tpl'}
                 {/block}
 
                 {block name='product_add_to_cart'}
                   {include file='catalog/_partials/product-add-to-cart.tpl'}
                 {/block}
+                
+                {block name='product_refresh'}
+                  <input class="product-refresh" data-url-update="false" name="refresh" type="submit" value="{l s='Refresh' d='Shop.Theme.Actions'}" hidden>
```

Comment:
Yes, the JS file checks the existence of this input in the DOM in order to launch the auto refresh, but the customer won't need to interact with it directly.